### ISSUE TITLE
Some elements of legacy skins shouldn't use animations when the skin provides its own

### ISSUE THREAD

**Issue by LittleEndu** [ppy/osu#6420]
I noticed this on judgements, but other elements that weren't animated by default on stable shouldn't use animations that are new in lazer. There might also be a special case for hit0.png files because they have animations in stable, but skin ones overwrite default ones.

**Comment by iiSaLMaN** [comment]
I think it's best to let legacy skin elements have their own animation instead of using lazer's.

**Comment by peppy** [comment]
Please provide specific detail and video comparisons of what you are seeing. What you describe is not specific enough to attain what you want fixed.

**Comment by LittleEndu** [comment]
I don't have any way to record and I'm not 100% sure of what I'm seeing (as I barely get 60fps)

On stable judgments don't have any transformation or fade animations if the skin has its own (the only animation being the fade out using the last frame skin provides). You can see it best on skins that animate hit0.png as doing so will overwrite stable's default. The same should apply to lazer - legacy skins that provide animation frames shouldn't use default animations.

**Comment by LittleEndu** [comment]
I am curious what is still missing in the details of this issue? Lazer adds additional animations to skins elements (like transform in), stable doesn't. Thus legacy skins shouldn't use lazer's additional animations.

**Comment by bdach** [comment]
As you've stated previously:

> I don't have any way to record and I'm not 100% sure of what I'm seeing (as I barely get 60fps)

I'm not saying you're wrong about what you're seeing but without specific examples of what you mean (which you've *vaguely* provided one of just now), we can only guess what you mean precisely, which is kind of counterproductive overall.

In general a catch-all and vague issue like this one isn't what we're particularly looking for. See other issues that have been opened in this repository for better examples (ie. #7584, #7392 - while probably not strictly related to what you are referring to, they're very specific issues that we don't have to narrow down or guess the meaning of).

**Comment by LittleEndu** [comment]
The 60fps issue has been remedied (stable has some weird settings for performance monitoring, all of which I had enabled) and I also now have a 144hz monitor. That makes it all the more noticeable that judgments stay in place on stable and just use the skins animation frames while they pop in on lazer.

I made this issue catch-all sounding because it feels like it would have a catch-all fix. And I don't know how many elements there are that are not animated in stable but are in lazer, that's why I said "Some elements".
